rxkata/src/test/java/org/sergiiz/rxkata/CountriesServiceSolvedTest.java

```diff
@@ -164,4 +165,56 @@ public void rx_MapCountriesToNamePopulation() {
         values.assertResult(expected);
         values.assertNoErrors();
     }
+
+    @Test
+    public void rx_CombineTwoObservablesAndReturnSumOfPopulation(){
+        // hint: use "map" operator
```

Comment:
Please rename all tests according to the following pattern: rx_<Tested_Method_Name> with behaviour explanation divided by `_`, see other test cases names as example